File: examples/plugin-archives/demo-multi-output-status/js/provider.js
function execute(params, upstream, context, api) {
  const rows = Math.max(1, Math.trunc(Number(params.rows ?? 5)));
  const scale = Number(params.scale ?? 2);
  const label = String(params.label ?? "sample");
  const table = api.Table.fromRecords(Array.from({ length: rows }, (_, index) => ({ index, value: index * scale })));
  return { outputs: { table, count: rows, label }, tableResult: table, plotResult: null, exportResult: null };
}

File: examples/plugin-archives/demo-multi-output-status/python/provider.py
def execute(params, upstream, context):
    import pandas as pd
    rows = max(1, int(params.get("rows", 5)))
    scale = float(params.get("scale", 2))
    label = str(params.get("label", "sample"))
    table = pd.DataFrame({"index": list(range(rows)), "value": [index * scale for index in range(rows)]})
    return {"outputs": {"table": table, "count": rows, "label": label}, "tableResult": table}
